lib/crowbar/client/app/upgrade.rb

```diff
@@ -219,12 +219,12 @@ def backup(component)
           catch_errors(e)
         end
 
-        desc "repocheck ADDON",
-          "Check for existing repositories for an addon (ha|storage)"
+        desc "repocheck COMPONENT",
+          "Check for existing repositories for a component (crowbar|ha|storage)"
```

Comment:
_crowbar_ is a bit confusing here. I understand it calls a method from `crowbar` module, but to user, this should be shown as a check for Cloud product repository